pallets/nfts/src/lib.rs

```diff
@@ -693,6 +736,24 @@ pub mod pallet {
 		CollectionNotEmpty,
 		/// The witness data should be provided.
 		WitnessRequired,
+		/// The account owns zero items in the collection.
+		NoItemOwned,
+		/// The collection has existing approvals.
+		CollectionApprovalsExist,
+		/// Collection and item approval conflicts.
+		DelegateApprovalConflict,
```

Comment:
Note: Tested in 
- cancel_approval_works_with_force: https://github.com/r0gue-io/pop-node/pull/387/files#r1881252586
- cancel_approval_works_with_admin: https://github.com/r0gue-io/pop-node/pull/387/files#r1881255136
- cancel_approval_works: https://github.com/r0gue-io/pop-node/pull/387/files#r1881257348
- clear_all_transfer_approval_works: https://github.com/r0gue-io/pop-node/pull/387/files#r1881256007